docs/toilAPI.rst

```diff
@@ -21,7 +21,7 @@ The Runner contains the methods needed to configure and start a Toil run.
 
 .. autoclass:: toil.job::Job.Runner
    :members:
-   
+
 Toil
```

Comment:
I can't reproduce this on my machine - can you send me a screenshot?

I think there are some formatting improvements to be made in that section, though, so I'll work on that.